options/options.go

```diff
@@ -165,11 +171,20 @@ func (o Options) ValidateRequired() error {
 		return fmt.Errorf("missing required option(s): %v", missingRequiredOptions)
 	}
 
-	if len(o.ProjKey) > maxProjKeyLength {
-		if strings.HasPrefix(o.ProjKey, "sdk-") {
-			return fmt.Errorf("provided projKey (%s) appears to be a LaunchDarkly SDK key", "sdk-xxxx")
-		} else if strings.HasPrefix(o.ProjKey, "api-") {
-			return fmt.Errorf("provided projKey (%s) appears to be a LaunchDarkly API access token", "api-xxxx")
+	if len(o.ProjKey) > 0 && len(o.Projects) > 0 {
+		return fmt.Errorf("`--projKey` cannot be combined with `projects` in configuration")
+	}
+
+	if len(o.ProjKey) > maxProjKeyLength && len(o.Projects) == 0 {
```

Comment:
remove `&& len(o.Projects) == 0` if `o.ProjKey` is less than maxProjKeyLength it shouldn't matter if projects is not empty.